lib/hacks/grid-area.js

```diff
@@ -0,0 +1,72 @@
+const parser = require('postcss-value-parser');
+
+const Declaration = require('../declaration');
+const translateShorthand = require('./grid-shorthand');
```

Comment:
If you use it only in the single file, it will be better to put `./grid-shorthand.js` content to `grid-area.js`.